waku/v2/protocol/filter/filter_test.go

```diff
@@ -29,6 +29,9 @@ func TestFilterSuite(t *testing.T) {
 	suite.Run(t, new(FilterTestSuite))
 }
 
+const defaultPubSubTopic = "/waku/2/go/filter/test"
```

Comment:
Better to rename to `defaultTestPubsubTopic` to avoid confusion. There is already a defaultpubsubTopic as "/waku/2/default-waku/proto" in pubsubtopic.go .

Reply:
Done at https://github.com/waku-org/go-waku/pull/916/commits/c076e36cec70d43bf80266b6ae33b175ce655860